samples/local_attestation/common/attestation.cpp

```diff
@@ -125,84 +161,97 @@ bool Attestation::attest_local_evidence(
         &claims_length);
     if (result != OE_OK)
     {
-        TRACE_ENCLAVE(
-            "oe_verify_evidence failed (%s).\n", oe_result_str(result));
+        TRACE_ENCLAVE("oe_verify_evidence failed (%s)", oe_result_str(result));
         goto exit;
     }
 
-    TRACE_ENCLAVE("oe_verify_evidence succeeded\n");
+    TRACE_ENCLAVE("oe_verify_evidence succeeded");
 
-    // Iterate through list of claims.
-    for (size_t i = 0; i < claims_length; i++)
+    // 2) validate the enclave identity's signed_id is the hash of the public
+    // signing key that was used to sign an enclave. Check that the enclave was
+    // signed by an trusted entity.
+
+    // Validate the signer id.
+    if ((claim_value =
+             _find_claim(claims, claims_length, OE_CLAIM_SIGNER_ID)) == NULL)
```

Comment:
I like the _find_claim() in remote_attestation better because it returns the value and the size. In this case, we assume the claim_value size to be 32 but it may not be true.

Reply:
Good point.